zaas-client/src/main/java/org/zowe/apiml/zaasclient/service/ZaasClient.java

```diff
@@ -106,6 +107,21 @@ public interface ZaasClient {
      */
     ZaasToken query(HttpServletRequest request) throws ZaasClientException;
 
+    /***
+     * Return details about the provided OIDC token by validating it against an API Mediation Layer instance.
+     *
+     * This method supports simple boolean validation against Zowe v2 and v2 as of v3.0
+     *
+     * A successful validation means the token was created using the same settings as found in the target API Mediation Layer.
+     * This version does not validate if the OIDC token is mapped to a user if the API Mediation Layer is running on the z platform.
```

Comment:
What does the mean exactly?  If a token is mapped to a mainframe user, `isValid()` will be false?   Does a service still have access to the OIDC token if the user is mapped?

Reply:
What I wanted to clarify with this is the fact that currently the API ML will say:
- `isValid()`: `true` if the OIDC token queried by the Zaas Client was issued by the OIDC provider that is configured in the queried API ML instance. The current implementation in the server side doesn't check if the token is mapped.
- `isValid()`: `false` if the OIDC token is not generated by the OIDC provider or if API ML is not enabled to handle OIDC tokens.

Regarding whether a service has access to the OIDC token if the user is mapped or not, I think it's unrelated to this change, as this simply enables services using the ZaasClient to query. So it will depend on the service's setup.

Reply:
Got it thanks.

> I think it's unrelated to this change, as this simply enables services using the ZaasClient to query.

It is unrelated.  I was merely curious.